product_docs/docs/pgd/5/admin-tpa/upgrading_major_rolling.mdx

```diff
@@ -438,20 +438,20 @@ Next, execute your test suite, including application transactions, queries, and
 
 ### Clean up post-upgrade
 
-We want to run a vacuum over to the database at this point as best practice. When the upgrade ran, you may have noticed the post-upgrade report included:
+As best practice, run a vacuum over the database at this point. When the upgrade ran, you may have noticed the post-upgrade report included:
```

Comment:
Was "over to" an error? It looked like it but my edit could be wrong...

Reply:
it was an error yes :-)